.github/workflows/release.yaml

```diff
@@ -126,21 +142,24 @@ jobs:
           tag: ${{ github.ref }}
       
       - name: Push manifests to GHCR with Flux
+        env:
+          CR_PAT_ARTIFACTS:  ${{ secrets.CR_PAT_ARTIFACTS }}
         run: |
             set -e
             cd config/.release/ && \
             flux push artifact oci://ghcr.io/${{ github.repository_owner }}/manifests/kyverno:$(git rev-parse --short HEAD) \
               --path="." \
               --source="$(git config --get remote.origin.url)" \
               --revision="(git tag --points-at HEAD)/$(git rev-parse HEAD)"
-              --creds flux:${{ secrets.CR_PAT_ARTIFACTS }}
+              --creds flux:${{env.CR_PAT_ARTIFACTS}}
```

Comment:
Did it fix the publishing issue ?

Reply:
not sure, need to test this😔

Reply:
Looks like a regular env var now. Shouldn't be `${{CR_PAT_ARTIFACTS}}` ?

Reply:
yes, changing.

Reply:
done!